phpunit/blocks/render-block-post-terms-test.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * File block rendering tests.
```

Comment:
Wrong header here.

Reply:
Fixed in [0469a23](https://github.com/WordPress/gutenberg/pull/59787/commits/0469a23c704228ea6979b80e2cb45dab38d09e7c)